module/attribute/src/Domain/Event/Attribute/AttributeTextareaParameterChangeEvent.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Attribute\Domain\Event\Attribute;
+
+use Ergonode\EventSourcing\Infrastructure\DomainEventInterface;
+use Ergonode\SharedKernel\Domain\Aggregate\AttributeId;
+use JMS\Serializer\Annotation as JMS;
+
+/**
+ */
+class AttributeTextareaParameterChangeEvent implements DomainEventInterface
```

Comment:
If you use name that are corresponding to Textarea attribute, that's be nice add with parameter is changing or if you add event for work with specific parameter type use type i name: 'AttributeBoolParameterChangeEvent'